### PR TITLE
blog/: add rte new revision post with resources

### PR DESCRIPTION
**Pull request by artur-rs**
Signed-off-by: Artur Raglis <artur.raglis@3mdeb.com>

### CODE REVIEW

**Comment by artur-rs** [comment]
scheduled on 21.06.2019

File: blog/content/post/2019-06-21-rte-new-version-and-kit.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+## Intro
+
+Today we are happy to announce the new version of Remote Testing Environment
+product - welcome RTE `v1.1.0`!
+
+After using RTE over a year at 3mdeb in everyday tasks regarding validation and
+developing firmware for various platforms, we have noticed more and more missing
+features in the presented Orange Pi Hat. Furthermore, we have received valuable
+feedback from our previous customers and we have addressed all requested
```
Comment (lwcislo) [inline blog/content/post/2019-06-21-rte-new-version-and-kit.md]: Zmieniłbym `previous customers` na coś w present perfect, bo takie sformułowanie daje asumpt do założeń, że oni z nami już nie współpracują.

File: blog/content/post/2019-06-21-rte-new-version-and-kit.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+![reset-button](/img/rte-reset-btn.jpg)
+
+* added `Open hardware` icon on PCB board
+
+![open-hardware-pcb](/covers/rte-new-revision.jpg)
+
+## Upgraded full kit
+
+We came to the conclusion to remove shop kit options for RTE. From now on there
```
Comment (lwcislo) [inline blog/content/post/2019-06-21-rte-new-version-and-kit.md]: "We came to the conclusion to remove" Jakoś fleksja mi nie pasuje. Może "We came to the conclusion that shop kit options for RTE are obsolete and should be removed."

**Comment by pietrushnic** [comment]
@lwcislo this is public repo and users seem to watch it so please use English.

File: blog/content/post/2019-06-21-rte-new-version-and-kit.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
+---
+title: RTE v1.1.0 - enhanced remote testing now available!
+abstract: Get familiar with new RTE revision - what has been changed, which
+          features are now more complex and which items has been added to the
```
Comment (pietrushnic) [inline blog/content/post/2019-06-21-rte-new-version-and-kit.md]:
```suggestion
          features are now more sophisticated and which items have been added to the
```

@artur-rs I'm not sure what you mean by `complex`, but let's not scare potential buyer :)

**Comment by lwcislo** [comment]
@pietrushnic  Of course. I haven't noticed it's public.

File: blog/content/post/2019-06-21-rte-new-version-and-kit.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+![reset-button](/img/rte-reset-btn.jpg)
+
+* added `Open hardware` icon on PCB board
+
+![open-hardware-pcb](/covers/rte-new-revision.jpg)
+
+## Upgraded full kit
+
+We came to the conclusion to remove shop kit options for RTE. From now on there
+is only one possible set to buy - the full one with all required elements for
```
Comment (pietrushnic) [inline blog/content/post/2019-06-21-rte-new-version-and-kit.md]: @artur-rs maybe link to shop?

File: blog/content/post/2019-06-21-rte-new-version-and-kit.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+## Summary
+
+Don't hesitate and check out our **RTE** product on the [3mdeb-shop](https://shop.3mdeb.com/product/rte/)
+where you can find more information about Orange Pi Zero shield or read about
+products related to platform security such as **Trusted Platform** modules.
+
+If you think that RTE is the product that You are looking for, but missing
+something crucial for your project or that we can help in improving the security
+of your firmware or you looking for someone who can boot your product by
```
Comment (pietrushnic) [inline blog/content/post/2019-06-21-rte-new-version-and-kit.md]:
```suggestion
of your firmware or you looking for someone who can boost your product by
```

File: blog/content/post/2019-06-21-rte-new-version-and-kit.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+leveraging advanced features of used hardware platform, feel free to
+[boot a call with us](https://calendly.com/3mdeb/consulting-remote-meeting)
```
Comment (pietrushnic) [inline blog/content/post/2019-06-21-rte-new-version-and-kit.md]:
```suggestion
[book a call with us](https://calendly.com/3mdeb/consulting-remote-meeting)
```
@artur-rs this should be changed in template

Comment (artur-rs) [reply blog/content/post/2019-06-21-rte-new-version-and-kit.md]: It is already fixed, I have written this post on the previous template

**Comment by artur-rs** [comment]
@pietrushnic all requested changes have been addressed, approval is needed for merging pull request

**Pull request merged**